src/nft/components/card/icons.tsx

```diff
@@ -37,17 +37,29 @@ const ListPriceRowContainer = styled(Row)`
 
 export const MarketplaceContainer = ({
   isSelected,
+  quantitySelected = 0,
   marketplace,
   tokenType,
   listedPrice,
   hidePrice,
 }: {
   isSelected: boolean
+  quantitySelected?: number
   marketplace?: Markets
   tokenType?: NftStandard
   listedPrice?: string
   hidePrice?: boolean
 }) => {
+  if (quantitySelected > 0) {
```

Comment:
prefer guard-style returns, like:

```
if (quantitySelected === 0) return 

return (
   <StyledMarketplaceContainer> ...
)

Reply:
this really isn't applicable we only care here if quantity selected is passed if so we render it - if not it goes to the other conditional checks below it